### PR TITLE
Added a check on the `alpha` argument

### PR DESCRIPTION
**Pull request by jgrancher**
Added a check on `hexAlpha`'s second argument `alpha`, so that:

```javascript
hexAlpha("#fa6d01", "");
hexAlpha("#fa6d01", false);
hexAlpha("#fa6d01", -1);
hexAlpha("#fa6d01", 2.4);
```

will all ignore it and return: `rgb(250,109,1)`.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ yarn add hex-alpha
 import hexAlpha from "hex-alpha";
 
 hexAlpha("#fa6d01", 0.1);
-// returns "rgba(250,109,1, 0.1)"
+// returns "rgba(250,109,1,0.1)"
```
Comment (jgrancher) [inline readme.md:23]: Small typo here.

File: utils.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * @param {string} input - A color in hexadecimal syntax. e.g. #fa6d01
  * @returns { R: string, G: string, B: string}
  */
-const hexToRGB = input => {
+const hexToRGB = (input) => {
```
Comment (jgrancher) [inline utils.js:5]: This whole file changes is my `prettier` doing his thing... If you don't want them, happy to roll them back!

Comment (nathsimpson) [reply utils.js:5]: Great. Thanks 👍

**Review by nathsimpson — APPROVED** [review]
LGTM. Thanks for the contribution! 🚀

**Pull request merged**